network-api/networkapi/wagtailpages/pagemodels/youtube.py

```diff
@@ -146,3 +146,20 @@ def get_context(self, request):
     class Meta:
         verbose_name = "YouTube Regrets 2021 Page"
         verbose_name_plural = "YouTube Regrets 2021 Pages"
+
+
+class YoutubeRegretsReporterExtensionPage(FoundationMetadataPageMixin, Page):
+
+    template = 'wagtailpages/pages/youtube_regrets_reporter_extension.html'
+    max_count = 1
+    zen_nav = True
+
+    content_panels = Page.content_panels  # Needed for wagtail-modeltranslation to work
+
+    def get_context(self, request):
+        context = super().get_context(request)
+        return set_main_site_nav_information(self, context, 'Homepage')
+
+    class Meta:
+        verbose_name = "YouTube Regrets Report Extension Page"
+        verbose_name_plural = "YouTube Regrets Report Extension Page"
```

Comment:
We don't need this `Meta` class: the verbose name is based on the classname (with spaces inserted) so that's alreayd fine, and if `max_count` is 1, then the plural name is irrelevant =)